etc/openstack-config/openstack-config.yml

```diff
@@ -1,60 +1,192 @@
 ---
 ###############################################################################
-# Configuration of OpenStack projects and users user environment.
```

Comment:
Usually we keep the existing comments and add the config in the relevant places. It doesn't really matter though.